bcipy/acquisition/README.md

```diff
@@ -30,23 +30,32 @@ The daq/tests/client_test.py file also demonstrates how the various components i
 
 ### Client Architecture
 
-A `DataAcquisitionClient` is initialized with a `Device`  and optionally a `Processor`, `Buffer`, and `Clock`. A device is a driver that knows how to connect to and communicate with specific EEG hardware/software, as well as decode the sensor data. Supported devices can be queried through the `registry`.
+A `DataAcquisitionClient` is initialized with a `Connector`  and optionally a `Processor`, `Buffer`, and `Clock`. A connector is a driver that knows how to connect to and communicate with specific EEG hardware/software, as well as decode the sensor data. Supported connectors can be queried through the `registry`.
 
 A `DataAcquisitionClient` manages two threads, one for acquisition and one for processing. The acquisition thread continually receives data and writes it to a process queue with an associated timestamp from the `clock`. The process thread watches the queue and sends data to the processor, as well as storing it in the `Buffer` to be queried and archived.
 
 #### Registry
 
-The registry has a list of supported devices. Devices can be queried by name. The device constructor is returned.
+The registry has a list of supported connectors, which can be queried by device and connection method.
 
     import daq.protocols.registry as registry
-    Device = registry.find_device('DSI')
+    registry.find_connector(device_spec=supported_device('DSI'), 
+                            connection_method=ConnectionMethod.TCP)
+
+
+#### Connector
```

Comment:
Sorry for the novice question, but what type of connector we are using in the system? Should we give an example here, or would defining params in configure_connector be enough?

Reply:
@celikbasak, it's helpful to know which parts of the documentation are unclear. I updated the README with an additional paragraph for currently supported Connectors. Hopefully this helps.